server/lib/config/pbench-server-default.cfg

```diff
@@ -128,17 +128,37 @@ lowerbound = 820
 # uri = driver://user:password@hostname/dbname
 wait_timeout = 120
 
+[authentication]
+# Secret key for use by the Pbench Server API sub-system for security related
+# tasks.
+secret-key = my_precious
```

Comment:
This should probably be commented. We don't want a *real* default `secret-key` of "my_precious"!

Reply:
Yeah, this should be a custom user-provided value and can be part of the `pbench-server.cfg` file.

Reply:
Addressed in PR #3081.

Reply:
Fixed.